app/screens/settings/section.js

```diff
@@ -33,7 +34,7 @@ function section(props) {
                     id={headerId}
                     defaultMessage={headerDefaultMessage}
                     values={headerValues}
-                    style={style.header}
+                    style={{...style.header, ...headerStyle}}
```

Comment:
use an array instead, if `headerStyle` is undefined or null this will cause a crash